web/packages/teleport/src/services/sales/index.ts

```diff
@@ -26,6 +26,8 @@ const UPGRADE_TEAM_URL = 'https://goteleport.com/r/upgrade-team';
 const UPGRADE_COMMUNITY_URL = 'https://goteleport.com/r/upgrade-community';
 // UPGRADE_IGS_URL is enterprise upgrading to enterprise with Identity Governance & Security
 const UPGRADE_IGS_URL = 'https://goteleport.com/r/upgrade-igs';
+// UPGRADE_POLICY_URL is enterprise upgrading to enterprise with Policy
+export const UPGRADE_POLICY_URL = 'https://goteleport.com/r/upgrade-policy';
```

Comment:
Added in https://github.com/gravitational/next/pull/2505

Reply:
We only need to change the redirect destination in `next`, the URLs in the frontend can stay, right?